lib/build.rb

```diff
@@ -66,7 +97,10 @@ def query_branches
             login = branch_details.commit.author.login
           end
 
-          @potential_builds << PotentialBuild.new(@client, @token, @repository, nil, b.commit.sha, b.name, login, nil, nil, nil, nil, nil)
+          # while looking through repo-branches, check if there is a PR associated and give it a PR num if so
+          associated_pr_num = branch_to_pr_number.fetch(b.name, nil)
```

Comment:
And here is where we attempt to pass the associated PR number. into the PotentialBuild constructor.  Note that if the branch is from NREL/EnergyPlus, but doesn't have an open PR yet, then this won't have a number and will return nil.  In this case, Decent CI will still write the status messages on the commit itself, as it has always done.